src/test/xattr_bench.cc

```diff
@@ -152,15 +152,6 @@ int main(int argc, char **argv) {
   argv_to_vec(argc, (const char **)argv, args);
 
   global_init(0, args, CEPH_ENTITY_TYPE_CLIENT, CODE_ENVIRONMENT_UTILITY, 0);
-  common_init_finish(g_ceph_context);
```

Comment:
@tchaikov  I checked we are using "g_ceph_context" only for accessing "filestore_xattr_use_omap" and now we are removing it so thought no need to initialize it.